build.gradle

```diff
@@ -193,11 +193,11 @@ task javaPreCommit() {
 }
 
 task javaPostCommit() {
-  dependsOn ":javaPreCommit"
+//  dependsOn ":javaPreCommit"
   dependsOn ":beam-runners-google-cloud-dataflow-java:postCommit"
   dependsOn ":beam-sdks-java-io-google-cloud-platform:postCommit"
-  dependsOn ":beam-sdks-java-extensions-sql:postCommit"
-  dependsOn ":beam-sdks-java-extensions-sql-jdbc:postCommit"
+//  dependsOn ":beam-sdks-java-extensions-sql:postCommit"
+//  dependsOn ":beam-sdks-java-extensions-sql-jdbc:postCommit"
```

Comment:
We want to revert these, right?

Reply:
Yes, all these build file changes introduced by commit https://github.com/apache/beam/pull/6261/commits/324714570d36c96ce85811ddd55d92730c6d17c8 will be reverted.